app/guid-node/metadata/-components/metadata-cedar/template.hbs

```diff
@@ -0,0 +1,22 @@
+<div local-class='metadata-cedar-container'>
+    <div local-class='header-container'>
+        <h2 data-test-schema-name>
+            {{t 'metadata.add-flow.add-title' schemaName=@template.schemaName }}
+        </h2>
+    </div>
+    <div local-class='cedar-container'>
+        <CedarMetadataEditor
+            @templateObject={{@template.cedarId}}
```

Comment:
```suggestion
            @templateObject={{@template.template}}
```

Reply:
This should work when we have the correct JSON for `@template.template` in mirage fixtures.